.circleci/config.yml

```diff
@@ -14,13 +14,19 @@ jobs:
           command: script/bootstrap
       - run:
           name: Update
-          command: script/update --clean -t $TARGET_ARCH
+          command: script/update --clean -t $TARGET_ARCH --use-bundled-sccache
+      - run:
+          name: Check sccache stats before build
+          command: tools/sccache/0.2.6/linux/sccache -s
```

Comment:
Maybe we can add a wrapper script to run bundled `sccache`? Like `script/sccache` that would run bundled `sccache` for current platform?
These absolute paths with versions and platforms in them look difficult to maintain.